.github/PULL_REQUEST_TEMPLATE

```diff
@@ -5,16 +5,14 @@ This PR ...
 ## List of changes
 
 - Change 1
-- Change 2
 
 ## Link to issue
 
-Closes `(replace this with "#<issue_number>" without the double-quotes)`
```

Comment:
Before this gets merged we should probably pull in the pull template from main

Reply:
Don't worry about it. Merging will fix this